pkg/console/util.go

```diff
@@ -409,6 +409,9 @@ func doInstall(g *gocui.Gui, hvstConfig *config.HarvesterConfig, webhooks Render
 		return err
 	}
 	defer os.Remove(cosConfigFile)
+	
+	// save current scheme version to harvester.config
+	hvstConfig.SchemeVersion = config.SchemeVersion
```

Comment:
Who will maintains the schemeVersion? if users can modifiy it how can we verify it accoridngly?

Reply:
Once we change the config struct, we should increase the scheme version.
and users shouldn't change it on their own.
It will be referred to by the future migration script.
Currently, we could not verify it if users change it manually.

Reply:
Please add an annotation.